libs/explorers/services/src/lib/comparison-tool-helper.service.ts

```diff
@@ -1,15 +1,26 @@
-import { Injectable } from '@angular/core';
+import { inject, Injectable } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
 import {
   ComparisonToolConfig,
   ComparisonToolConfigColumn,
   ComparisonToolLink,
   HeatmapCircleData,
+  VisualizationOverviewPane,
 } from '@sagebionetworks/explorers/models';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ComparisonToolHelperService {
+  private readonly sanitizer = inject(DomSanitizer);
+
+  createVisualizationOverviewPane(heading: string, htmlContent: string): VisualizationOverviewPane {
+    return {
+      heading,
+      content: this.sanitizer.bypassSecurityTrustHtml(htmlContent),
```

Comment:
Visualization Overview panes use html in the content so they must be sanitized

Reply:
`bypassSecurityTrustHtml` short-circuits Angular’s sanitizer, so our dialog’s `[innerHtml]` binding never scrubs the overview content. Since these panes only use basic tags/`<img>` (nothing the sanitizer would drop), we can safely return the raw string and let Angular sanitize at render time:

```suggestion
      content: htmlContent,
```

We would also need to update the `VisualizationOverviewPane` interface so that `content` is a `string` rather than `SafeHtml`: 

```
export interface VisualizationOverviewPane {
  heading: string;
  content: string;
}
```

After these changes, the existing `[innerHtml]="pane.content"` call will pick up Angular’s default sanitization again.

Reply:
Good idea.  The original implementation used it but we don't need this functionality.